javascript/javascript1/week1/housey pricey/housey-pricey.js

```diff
@@ -0,0 +1,21 @@
+
+
+name = "Peter";
+const volumeInMetersPeter = (8 * 10 * 10);
```

Comment:
Even though this is mathematically correct then I would abstract each of the values out as:
`const widthInMetersPeter = 8`
`const depthInMetersPeter = 10`
`const heightInMetersPeter = 10`

Then calculating the volume in meters:
`const volumeInMetersPeter = (widthInMetersPeter * depthInMetersPeter * heightInMetersPeter);`

Being as explicit as possible and assigning values you use helps you in the future when you have to go back to old code bases. It also helps if other people were to review your code!